pkg/ibmvpc/analysis_output_test.go

```diff
@@ -69,32 +70,34 @@ type vpcGeneralTest struct {
 }
 
 const (
-	actualOutFilePrefix            = "out_"
-	inputFilePrefix                = "input_"
-	suffixOutFileWithGrouping      = "_with_grouping"
-	suffixOutFileDebugSubnet       = "_analysisPerSubnetSeparately"
-	suffixOutFileSubnetsLevel      = "subnetsBased_withPGW"
-	suffixOutFileSubnetsLevelNoPGW = "subnetsBased_withoutPGW"
-	suffixOutFileDiffSubnets       = "subnetsDiff"
-	suffixOutFileDiffEndpoints     = "endpointsDiff"
-	suffixOutFileExplain           = "explain"
-	txtOutSuffix                   = ".txt"
-	debugOutSuffix                 = "_debug.txt"
-	mdOutSuffix                    = ".md"
-	jsonOutSuffix                  = ".json"
-	secJSONOutSuffix               = "_2nd.json"
-	drawioOutSuffix                = ".drawio"
-	archDrawioOutSuffix            = "_arch.drawio"
-	svgOutSuffix                   = ".svg"
-	archSvgOutSuffix               = "_arch.svg"
-	htmlOutSuffix                  = ".html"
-	archHTMLOutSuffix              = "_arch.html"
+	actualOutFilePrefix               = "out_"
+	inputFilePrefix                   = "input_"
+	suffixOutFileWithGrouping         = "_with_grouping"
+	suffixOutFileWithoutLbAbstraction = "_no_lbAbstract"
+	suffixOutFileDebugSubnet          = "_analysisPerSubnetSeparately"
+	suffixOutFileSubnetsLevel         = "subnetsBased_withPGW"
+	suffixOutFileSubnetsLevelNoPGW    = "subnetsBased_withoutPGW"
+	suffixOutFileDiffSubnets          = "subnetsDiff"
+	suffixOutFileDiffEndpoints        = "endpointsDiff"
+	suffixOutFileExplain              = "explain"
+	txtOutSuffix                      = ".txt"
+	debugOutSuffix                    = "_debug.txt"
+	mdOutSuffix                       = ".md"
+	jsonOutSuffix                     = ".json"
+	secJSONOutSuffix                  = "_2nd.json"
+	drawioOutSuffix                   = ".drawio"
+	archDrawioOutSuffix               = "_arch.drawio"
+	svgOutSuffix                      = ".svg"
+	archSvgOutSuffix                  = "_arch.svg"
+	htmlOutSuffix                     = ".html"
+	archHTMLOutSuffix                 = "_arch.html"
 )
 
 // getTestFileName returns expected file name and actual file name, for the relevant use case
 func getTestFileName(testName string,
 	uc vpcmodel.OutputUseCase,
 	grouping bool,
+	noLbAbstract bool,
```

Comment:
Same here - consider removing the negation

Reply:
most common case is with abstraction, I uses negation so:
1. I wont have to add  "lbAbstract: true," to every test in the test list
2. wont need to add  "_lbAbstract" to every output file